nomad/integration-tests-new.nomad

```diff
@@ -51,9 +51,17 @@ variable "integration_tests_kafka_sasl_password" {
   description = "The Confluent Cloud API secret to configure integration test consumers with."
 }
 
-variable "rust_log" {
+variable "dns_server" {
   type        = string
-  description = "Controls the logging behavior of Rust-based services."
+  description = "The network.dns.server value. This should be equivalent to the host's ip in order to communicate with dnsmasq and allow consul dns to be available from within containers. This can be replaced as of Nomad 1.3.0 with variable interpolation per https://github.com/hashicorp/nomad/issues/11851."
+  default     = ""
+}
+
```

Comment:
seems like a weird thing to have snuck its way in 😬

Reply:
fixed